quickwit/quickwit-metastore/src/metastore/postgres/metastore.rs

```diff
@@ -1495,8 +1495,9 @@ impl MetastoreService for PostgresqlMetastore {
         const PRUNE_AGE_SHARDS_QUERY: &str = include_str!("queries/shards/prune_age.sql");
         const PRUNE_COUNT_SHARDS_QUERY: &str = include_str!("queries/shards/prune_count.sql");
 
-        if let Some(max_age) = request.max_age {
-            let limit_datetime = OffsetDateTime::now_utc() - Duration::from_secs(max_age as u64);
+        if let Some(max_age_secs) = request.max_age_secs {
+            let limit_datetime =
```

Comment:
same nitpick as above.

Reply:
In the comment above, it was possible to rewrite the calculations to use `Add` instead of `Sub`. Can we do something similar here or should we just explicit why we are confident that no underflow can happen?

Reply:
ah right. no pb.